src/main/java/com/minecolonies/entity/ai/basic/AbstractEntityAIBasic.java

```diff
@@ -121,47 +125,47 @@ protected AbstractEntityAIBasic(@NotNull final J job)
     {
         super(job);
         super.registerTargets(
-                /**
+                /*
                  * Init safety checks and transition to IDLE
                  */
-                new AITarget(INIT, this::initSafetyChecks),
-                /**
+          new AITarget(INIT, this::initSafetyChecks),
+                /*
                  * Update chestbelt and nametag
                  * Will be executed every time
                  * and does not stop execution
                  */
-                new AITarget(this::updateVisualState),
-                /**
+          new AITarget(this::updateVisualState),
+                /*
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Block comment has incorrect indentation level 16, expected is 10, indentation should be the same level as line 144. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.indentation.CommentsIndentationCheck)